dandischema/consts.py

```diff
@@ -1,6 +1,6 @@
 from packaging.version import Version as _Version
 
-DANDI_SCHEMA_VERSION = "0.6.10"
+DANDI_SCHEMA_VERSION = "0.6.11"
```

Comment:
The patch is boosted instead of minor here which contradicts with the top post, https://github.com/dandi/dandi-schema/pull/342#issue-3558387078, which indicates booting of minor.  However, "minor" can possible mean patch there.